configuration/logs.mdx

```diff
@@ -1,13 +1,17 @@
 ---
 title: "Logs"
-description: "Using `tracing` or `log` with Shuttle apps."
+description: "Tracing or logging in Shuttle apps."
 ---
 
-## Tracing
+When you build an app with Shuttle, a global subscriber will be created and installed behind the scenes. If you'd rather set up
+your own tracing or logging, you can opt-out of the default subscriber by disabling the default features on `shuttle-runtime`:
```

Comment:
I would add a note about `shuttle-next` too:

```
For `shuttle-next` projects the default global subscriber is installed by default and it can not be opted out.
```

Reply:
Good catch, thanks!